static/js/components/dashboard/CouponCard.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import React from 'react';
+import Card from 'react-mdl/lib/Card/Card';
+import CardTitle from 'react-mdl/lib/Card/CardTitle';
+
+import type { Coupon } from '../../flow/couponTypes';
+import CouponMessage from './CouponMessage';
+
+export default class CouponCard extends React.Component {
+  props: {
+    coupon: Coupon,
+  };
+
+  render() {
+    const { coupon } = this.props;
+
+    return <Card shadow={0}>
+      <CardTitle>Coupons</CardTitle>
+      <div>
+        <CouponMessage coupon={coupon} />
+      </div>
+    </Card>;
+  }
+}
```

Comment:
this strikes me as a nice component to declare using an arrow function - kind of just a style choice though.